package.json

```diff
@@ -23,22 +23,22 @@
     "external-services-offline": "docker compose --project-name hash-external-services-offline --file apps/hash-external-services/docker-compose.yml --env-file .env",
     "external-services-test": "docker compose --project-name hash-external-services-test --file apps/hash-external-services/docker-compose.yml --file apps/hash-external-services/docker-compose.type-fetcher.yml --file apps/hash-external-services/docker-compose.test.yml --env-file .env",
     "fix": "npm-run-all --continue-on-error \"fix:*\"",
-    "@todo.1": "Remove the ignore argument below for mock-block-dock once repo is upgraded fully to 0.2 spec",
+    "@TODO.1": "Restore dependency consistency for block packages and remove the ignore argument below",
+    "fix:deduplicate": "yarn install && yarn-deduplicate --strategy=fewer && yarn install",
```

Comment:
Are we actually dependent on alphabetical ordering... Please tell me that's not the case 😭 

If it truly is that, then I think we might want to consider polluting our `package.json` even more and creating duplicate scripts like 
- `lint-pipeline:0-deduplicate: lint:yarn-deduplicate`, 
- `lint-pipeline: 1-codegen: lint:estlint`
- `"lint": "npm-run-all --continue-on-error \"lint-pipeline:*\""`